src/main/java/net/minecraftforge/event/entity/living/EnderManAngerEvent.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.entity.monster.EnderMan;
+import net.minecraft.world.entity.player.Player;
+import net.minecraftforge.eventbus.api.Cancelable;
+
+/**
+ * This event is fired when an Enderman detects that a player is looking at them.
+ * It will not be fired if the detection is prevented by {@link IForgeItem#isEnderMask}
+ * <p>
+ * This event is {@link Cancelable}.
+ * If this event is canceled, the Enderman will not target the player.
+ * <p>
+ * This event does not have a {@link Result}.
```

Comment:
Probably worth mentioning which bus this fires on (as a quick and easy reference so that one doesn't have to follow through the event hierarchy looking for IModBusEvent) as well as which side(s) it fires on.